### ISSUE TITLE
[FORMATTING] TransactSQL incorrectly formats some block comments, leading to broken statements

### ISSUE THREAD

**Issue by wylieconlon** [sql-formatter-org/sql-formatter#747]
**Input data**

```sql
/** block comment **/
```

**Expected Output**

```sql
/** block comment **/
```

**Actual Output**

```sql
/ * * block comment * * /
```

**Usage**

This example reproduces on the sql-formatter playground. The only option that fixes the bug is enabling "dense operators", which produces this output:

```sql
/**block comment**/
```

**Comment by nene** [comment]
Thanks for reporting.

Looks like this bug is related to the parsing of nested block-comments. Affects Transact-SQL, Postgres, DB2 for IMBi.

The the final `**/` seems to trigger it:

```sql
/* block comment **/
```

results in:

```sql
/ * block comment ** /
```

**Comment by nene** [comment]
The fix for this is now out in 15.3.2
